core/engine/src/builtins/array/mod.rs

```diff
@@ -193,6 +193,9 @@ impl BuiltInObject for Array {
 }
 
 impl BuiltInConstructor for Array {
+    const P: usize = 41;
+    const SP: usize = 5;
```

Comment:
```suggestion
    const PPOPERTY_COUNT: usize = 41;
    const STATIC_PROPERTY_COUNT: usize = 5;
```

I think it would be better to add the full name instead of single letter.